### PR TITLE
Bug report with version command

### PR DESCRIPTION
**Pull request by gillespiecd**
Instruct users to run `version` command when filing a bug report, helps with diagnostics

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -23,8 +23,24 @@ A clear and concise description of what you expected to happen.
...
+ commit_hash        │ 21882e5d19eab7b8ed340ad38396234a15db7b35
+ commit_date        │ 2020-12-08T05:10:51+00:00
+ build_os           │ macos-x86_64
+ rust_version       │ rustc 1.48.0 (7eac88abb 2020-11-16)
+ rust_channel       │ stable-x86_64-apple-darwin (default)
+ cargo_version      │ cargo 1.48.0 (65cbdd2dc 2020-10-14)
+ pkg_version        │ 0.23.0
+ build_time         │ 2020-12-08 08:35:51
+ build_rust_channel │ debug
+ features           │ ctrlc, default, directories, dirs, ...
```
Comment (jonathandturner) [inline .github/ISSUE_TEMPLATE/bug_report.md:42]: I think by default `version` will print the table the other direction, which might be hard to copy/paste.

@fdncred - any thoughts?

Comment (fdncred) [reply .github/ISSUE_TEMPLATE/bug_report.md:42]: Ya, I have a custom command that I use now `def nuver [] { version | pivot key value }`

Comment (jonathandturner) [reply .github/ISSUE_TEMPLATE/bug_report.md:42]: Maybe we should tell them to run `version | pivot key value`

Comment (gillespiecd) [reply .github/ISSUE_TEMPLATE/bug_report.md:42]: Great point, updated to use `version | pivot`.

**Comment by jonathandturner** [comment]
Looks good!

**Pull request merged**